### DISCUSSION TITLE
Treasury Proposal to Incentive subapidao staking for providing low cost default Oracle of  widely used ORMP/ORMPPort

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1423] [General]
This is another initiative alternate to https://github.com/orgs/subapidao/discussions/48.

As subapidao need to provide high service level by requiring the subapi nodes to perform more reliable, they are going to  introducing RING staking mechanism and slash mechanism in the next version (eta 2 months). To achieve this, they will need to provide incentive for the staking by gathering more income or increase oracle price shortly.

Instead of forcing them to increase oracle price shortly, I think Darwinia Treasury is better provide the incentive for them at the startup and go-to-market period. The reasons is as below:

1. They are using RING for staking, this will increase the demand for RING.
2. Msgport is still in the go to market period, higher oracle price will lead to higher message price which may not good for the adoption of the cross-chain message users. So instead of increasing the oracle price for staking incentive, Treasury proposal can help on this.

For this reason, I think the subapidao are welcome to propose to Darwinia Treasury to allocate RINGs for its Node Staking incentive. Before acting, the automation and contract for staking is still in pending. cc @jiguantong 

Ref:

https://github.com/orgs/subapidao/discussions/48
https://github.com/orgs/subapidao/discussions/49

**Comment by jiguantong** [comment]
It's a good idea. But currently, the number of messages sent through Msgport is very low, so we cannot estimate the revenue and cannot determine how many RINGs to apply for.
I think we can first reduce the Oracle fees and then make a decision once automation staking is completed.
